pkgs/by-name/sc/scrutiny/package.nix

```diff
@@ -0,0 +1,77 @@
+{ buildNpmPackage
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+, lib
+, ...
+}:
+let
+  name = "scrutiny";
+  version = "0.7.2";
+
+  repo = fetchFromGitHub {
+    owner = "AnalogJ";
+    repo = name;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-UYKi+WTsasUaE6irzMAHr66k7wXyec8FXc8AWjEk0qs=";
+  };
+
+  frontend = buildNpmPackage rec {
+    inherit version;
+    pname = "${name}-webapp";
+    src = "${repo}/webapp/frontend";
+
+    npmDepsHash = "sha256-M8P41LPg7oJ/C9abDuNM5Mn+OO0zK56CKi2BwLxv8oQ=";
+
+    buildPhase = ''
+      runHook preBuild
+      mkdir dist
+      npm run build:prod --offline -- --output-path=dist
+      runHook postBuild
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      mkdir $out
+      cp -r dist/* $out
+      runHook postInstall
+    '';
+  };
+in
+buildGoModule rec {
+  inherit version;
+  pname = name;
+  src = repo;
+
+  subPackages = "webapp/backend/cmd/scrutiny";
+
+  vendorHash = "sha256-SiQw6pq0Fyy8Ia39S/Vgp9Mlfog2drtVn43g+GXiQuI=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [ "-extldflags=-static" ];
+
+  tags = [
+    "netgo"
+    "static"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin $out/share/scrutiny
+    cp $GOPATH/bin/scrutiny $out/bin/scrutiny
+    cp -r ${frontend}/* $out/share/scrutiny
```

Comment:
We could move the installation of the frontend into `postInstall`, so we don't have to overwrite the builders installPhase?

Reply:
Good idea! Thanks!